string topic
string node_uri
---
